dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/Spnego.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.websphere.simplicity.config;
+
+import javax.xml.bind.annotation.XmlAttribute;
+
+/**
+ * Represents the <kerberos> element in server.xml
+ */
+public class Spnego extends ConfigElement {
+
+    @XmlAttribute
+    public String krb5Keytab;
+
+    @XmlAttribute
+    public String krb5Config;
+
+    @XmlAttribute
+    public String servicePrincipalNames;
+
+    @XmlAttribute
+    public String canonicalHostName;
+
+    @XmlAttribute
+    public String skipForUnprotectedURI;
+
+    @XmlAttribute
+    public String trimKerberosRealmNameFromPrincipal;
+
+    @XmlAttribute
+    public String spnegoAuthenticationErrorPageURL;
+
+    @XmlAttribute
+    public String spnegoNotSupportedErrorPageURL;
+
+    @XmlAttribute
+    public String ntlmTokenReceivedErrorPageURL;
+
+    @Override
+    public String toString() {
+        StringBuilder buf = new StringBuilder(getClass().getSimpleName()).append('{');
```

Comment:
Typically in these toStrings for the jaxb classes, we don't print for values that are null.